packages/devtools_app/lib/src/shared/version.dart

```diff
@@ -2,21 +2,21 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// @dart=2.9
+// ignore_for_file: import_of_legacy_library_into_null_safe
 
 import 'package:flutter/material.dart';
 
 import 'utils.dart';
 
 class FlutterVersion extends SemanticVersion {
   FlutterVersion._({
-    @required this.version,
-    @required this.channel,
-    @required this.repositoryUrl,
-    @required this.frameworkRevision,
-    @required this.frameworkCommitDate,
-    @required this.engineRevision,
-    @required this.dartSdkVersion,
+    required this.version,
+    required this.channel,
+    this.repositoryUrl,
```

Comment:
why is this no longer required?

Reply:
We check it for null later in the code, so I assume it sometimes can be null

Reply:
fixed